src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -1649,6 +727,24 @@ describe("Decimal", () => {
       expect(hiddenValue()).toBe("");
     });
 
+    it("does not format a value that is not a number", () => {
+      const onBlur = jest.fn();
+      render({
+        value: "",
+        onBlur,
+        onChange: (e) => {
+          setProps({ value: e.target.value.rawValue });
+        },
+        allowEmptyValue: true,
+      });
+
+      setProps({ value: "FooBar" });
+      blur();
+      expect(value()).toBe("FooBar");
+      expect(hiddenValue()).toBe("FooBar");
```

Comment:
Should we update hiddenValue to be FooBar? Normally the expectation of the hiddenValue is that it is strictly a plainly formatted (i.e. only the seperator & numbers present) value. If the typed value is `FooBar` i'd expect the hiddenValue to be `null`

Reply:
My thoughts on this were that the consumer would be using the hiddenValue for validation purposes. I'm happy to change it to `null` to meet the expectation of it being a plainly formatted value though.

Reply:
I've just reviewed this @chrisbarber86 but in this case I think we should actually send `FooBar`. If we were to send a `POST` with nothing as the value the backend could only say `this field is required`, but not `FooBar isn't valid`.

If the field was optional, and we sent nothing the backend wouldnt be able to flag that the input value was discarded, which is the same issue we're trying to avoid in SPA validations.

Reply:
Ok - so the consumer on the front end would need to add validations if they don't want to send `FooBar`. otherwise if they don't it's on the server to handle that.

I'm ok with that.